src/dbg/debugger.cpp

```diff
@@ -581,7 +583,42 @@ static void cbGenericBreakpoint(BP_TYPE bptype, void* ExceptionAddress = nullptr
     if(*bp.commandText && commandCondition)  //command
     {
         //TODO: commands like run/step etc will fuck up your shit
-        DbgCmdExec(bp.commandText);
+        varset("$breakpointcondition", breakCondition ? 1 : 0, false);
+        varset("$breakpointlogcondition", logCondition, false);
+        _dbg_dbgcmddirectexec(bp.commandText);

```

Comment:
There isn't any ready-made script file that make use of script command to pause now. If we introduce script-controlled break condition earlier, then less people will make these script files. We should tell them to use "breakpointcondition" instead of such commands.